### ISSUE TITLE
[Godot 3.2 stable] Memory leak and crash when clicking on "Skeleton" node in scene.

### ISSUE THREAD

**Issue by Skalli** [godotengine/godot#35936]
tl;dr: When clicking on a skeleton node in certain scenes Godots editor keeps allocating memory until it runs out and crashes in mere seconds.

**Godot version:**
3.2.stable.mono.official

**OS/device:**
Arch Linux; 

64-bit.

**Issue description:**
I noticed this when working with models from Make Human, it is reproducible with the steps done in the next section. But whenever I click on the Skeleton node in the scene hierarchy Godot eats up all available memory, then all available swap until it finally crashes.

**How to reproduce:**
* Create any model with Make Human (v1.1.1 if that matters).
* Export as *.dae (other options might work too).
* Import the file in Blender (v2.81a).
* Export from Blender as *.gltf to a Godot directory.
* Open the gltf in Godot and select "New Inherited".
* Save scene as *.tscn.
* Watch as Godot is frozen, memory consumption goes through the sky and finally crashes as no memory is available anymore.
* ...
* Profit?

**Some pictures:**
![scene hierarchy](https://user-images.githubusercontent.com/1928826/73881105-0d92b480-4860-11ea-80b1-9cbd5b6c9298.png)
Click on Skeleton to cause memory leak.

![memory_leak](https://user-images.githubusercontent.com/1928826/73881156-20a58480-4860-11ea-8999-8e07614c1840.png)
Memory consumption in system monitor. Total is not correct because I killed the task to make the screenshot. Memory leak can take system down with it. It grows until it runs out of space.

Scene file used:
(Will attach shortly, right now it won't let me.)

**Comment by Calinou** [comment]
Duplicate of #35652.

**Comment by Skalli** [comment]
Ah, true, I searched for the terms, but this issue didn't showed up. Sorry!
